pages/mysymptoms/all.js

```diff
@@ -134,7 +142,7 @@ export default function AllSymptoms({ mySymptoms, handleSymptomDelete }) {
   } else {
     return (
       <>
-        <Header backButton title="Alle Symptome" />
+        <Header link="../mysymptoms" title="Alle Symptome" />
```

Comment:
This doesn't look right. For links you need to pass the path **starting from the root**. It's only for imports that you need to use the relative path.

Reply:
changed it :)